Mathlib/NumberTheory/SiegelsLemma.lean

```diff
@@ -23,34 +23,36 @@ system it determines has a non-zero integer solution `t` with
 
 ## Notation
 
- - `‖_‖ ` : Matrix.seminormedAddCommGroup is the sup norm, the maximum of the absolute values of
- the entries of the matrix
+- `‖_‖ ` : Matrix.seminormedAddCommGroup is the sup norm, the maximum of the absolute values of
+the entries of the matrix
 
 ## References
 
 See [M. Hindry and J. Silverman, Diophantine Geometry: an Introduction][hindrysilverman00].
 -/
 
-/-We set ‖⬝‖ to be Matrix.seminormedAddCommGroup  -/
+/- We set ‖⬝‖ to be Matrix.seminormedAddCommGroup  -/
 attribute [local instance] Matrix.seminormedAddCommGroup
 
 open Matrix Finset
 
 namespace Int.Matrix
 
-variable (m n : ℕ) (A : Matrix (Fin m) (Fin n) ℤ) (v : Fin n → ℤ) (hn : m < n) (hm : 0 < m)
+variable {α β : Type _} [Fintype α] [Fintype β] [DecidableEq β] [DecidableEq α] (m n : ℕ)
```

Comment:
Do you really need `m` and `n`? I mean, you can just have `Fintype.card ... < Fintype.card` and friends. The line about notations will be longer, but this is the only drawback I see.

Reply:
Good point! I got rid of them as variables and introduced as notation. This even simplified some lines later.
Thanks!